src/main/java/checkbot/Ui.java

```diff
@@ -0,0 +1,118 @@
+package checkbot;
+
+import checkbot.exception.CheckbotException;
+import checkbot.task.Task;
+import checkbot.task.TodoList;
+
+/**
+ * Handles printing of messages to the user.
+ */
+public class Ui {
+    public static final String INDENTATION = "  ";
+    private static final String SEPARATOR = "____________________________________________________________";
+
+    /**
+     * Shows the welcome message to the user.
+     */
+    public void showWelcome() {
+        print(
+                "Hello, I'm Checkbot, your personal assistant.\n" +
```

Comment:
[Break before the operator](https://se-education.org/guides/conventions/java/intermediate.html#layout)  (i.e. put the + on the next line)